adapter/pkg/config/log_types.go

```diff
@@ -48,8 +49,9 @@ type LogConfig struct {
 
 func getDefaultLogConfig() *LogConfig {
 	adapterLogConfig = &LogConfig{
-		Logfile:  "/dev/null",
-		LogLevel: "INFO",
+		Logfile:   "/dev/null",
+		LogLevel:  "INFO",
+		LogFormat: "TEXT",
```

Comment:
Let's use the constants you've defined in `adapter/pkg/logging/log_constants.go`

Reply:
`adapter/pkg/logging/log_constants.go` is inside `github.com/wso2/product-microgateway/adapter/pkg/logging` package and `adapter/pkg/logging/log_constants.go` has already imported the package `github.com/wso2/product-microgateway/adapter/pkg/config`.
In order to use the constants we would need to import `github.com/wso2/product-microgateway/adapter/pkg/logging` package inside `adapter/pkg/config/log_types.go`.
Then there would be a cycle import and it's not allowed..

Anyway, since this is a default config, I think using the hard coded value here would be ok, as other attributes do the same.